package.json

```
@@ -4,7 +4,9 @@
   "description": "Docs for SolidStart",
   "type": "module",
   "scripts": {
+    "compile": "tsc --noEmit",
```

Ooh per https://github.com/FullScreenShenanigans/EightBittr/issues/322 maybe this should be...

```suggestion
    "compile": "yarn tsc --noEmit",
```

...just to assume the user doesn't have the `typescript` package installed globally?

---

Ok I tried this locally after running `npm uninstall -g typescript` and it worked fine. Never mind!